provided.al2/graalvm/java11/cookiecutter-aws-sam-graalvm-gradle/{{cookiecutter.project_name}}/Dockerfile

```diff
@@ -1,5 +1,12 @@
+ARG PLATFORMARG=linux/amd64
+        ### Example: "linux/amd64", "windows/amd64", "linux/aarch64", "darwin/aarch64", ..  See supported values at: https://github.com/graalvm/graalvm-ce-builds/releases/
+### !! Attention !! Repeat this for the "ENV" line below for "PLATFORM" env-variable.
+
 FROM public.ecr.aws/amazonlinux/amazonlinux:2
 
+ENV PLATFORM linux-amd64
+        ### Attention: Unlike the "ARG" line at the top of this file, replace "/" with "-" (hyphen).
```

Comment:
Same here, can we remove the leading spaces here?

Reply:
Yup!  Thanks!